### PR TITLE
Add support for AWS EBS

### PR DESCRIPTION
**Pull request by rajikaimal**
## What did you implement:

Features listed in #17

## How can we verify it:

```
npm test
or
yarn test
```

## Todos:

- [ ] Make sure code coverage hasn't dropped

## Completed:

- [x] Write tests
- [x] Write documentation
- [x] Fix linting errors
- [x] Provide verification config / commands / resources
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

cc @rehrumesh @charithccmc

### CODE REVIEW

File: examples/storage/aws.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+console.log('creating ebs image');
+// create AWS ESB image
+ebs
+  .create(params)
+  .then(res => {
+    console.log(`All done ! ${res}`);
+  })
+  .catch(err => {
+    console.log(`Oops something happened ${err}`);
```
Comment (rehrumesh) [inline examples/storage/aws.js]: use console.error

File: examples/compute/google.js [hunk 2]
```diff
@@ -7,14 +7,24 @@ const ncGoogle = nodeCloud.getProvider("google", {
...
+//   .catch(err => {
+//     console.log(err);
+//   });
+
+gceCompute.list({
+      maxResults: 1
+    })
   .then(res => {
-    console.log(res);
+    // console.log(res);
```
Comment (rehrumesh) [inline examples/compute/google.js]: remove unnecessary console.logs and comments

**Comment by rehrumesh** [comment]
@rajikaimal please update the PR with codacy quality suggestions.

**Comment by rajikaimal** [comment]
@rehrumesh Fixed linting issues, `Codacy` is still reporting some issues in test files. Looks like it still haven't updated to the latest commit.

**Pull request merged**